spec/features/mentors/applications/index_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'Application index page', type: :feature do
+  let(:user)         { create(:user) }
+  let(:project)      { create(:project, :in_current_season, :accepted, submitter: user) }
+  let(:team1)        { create(:team, name: 'We could be Heroines') }
+  let(:team2)        { create(:team, name: 'Electric Queen') }
+  let(:application1) { create(:application, :in_current_season, :for_project, project1: project, team: team1) }
+  let(:application2) { create(:application, :in_current_season, :for_project, project2: project, team: team2) }
+
+  # TODO: use timecop to make sure the site is available during season
```

Comment:
Is this a TODO for this PR?

Reply:
Depends 😉 

I thought if we agree on feature specs being useful, I'd update the ones for the mentor bits of the app to actually test more than just the tiny changes from this PR and then also address the timing things there.

Reply:
Well, I added the time travel now. It's not really necessary for the specs to work - when I added the comment, I just thought it would make the specs more close to _"the real world™"_, since these bits of the app are only used during a certain phase of the season...

Do you think this makes sense? Or should I get rid of the time travel again to make the tests shorter / simpler?

Reply:
I don't mind either. If it's not required, might as well remove it. But then again, you already added it and it simulates our use case closer. `/shrug`